### DISCUSSION TITLE
Broken Bard API

### DISCUSSION THREAD

**Discussion by suryanshgupta9933** [dsdanielpark/Bard-API#98] [General]
Is the bardapi completely broken because of Google's internal changes and updated API policy? Is there any chance of it getting fixed anytime soon?

**Comment by dsdanielpark** [comment]
It still work. We update more features recently.

**Comment by RAHULFROST7** [comment]
i have tried recently this the **error i got** 

----> 4 bard = Bard(token=token)
      5 bard.get_answer(data)['content']

File c:\Users\Hp\AppData\Local\Programs\Python\Python310\lib\site-packages\bardapi\core.py:64, in Bard.__init__(self, token, timeout, proxies, session, conversation_id, google_translator_api_key, language, run_code, token_from_browser)
     62 else:
     63     self.session = session
---> 64 self.SNlM0e = self._get_snim0e()
     65 self.language = language or os.getenv("_BARD_API_LANG")
     66 self.run_code = run_code or False

File c:\Users\Hp\AppData\Local\Programs\Python\Python310\lib\site-packages\bardapi\core.py:474, in Bard._get_snim0e(self)
    472 snim0e = re.search(r"SNlM0e\":\"(.*?)\"", resp.text)
    473 if not snim0e:
--> 474     raise Exception(
    475         "SNlM0e value not found. Double-check __Secure-1PSID value or pass it as token='xxxxx'."
    476     )
    477 return snim0e.group(1)

Exception: SNlM0e value not found. Double-check __Secure-1PSID value or pass it as token='xxxxx'.

i have **given the __Secure-1PSID value properly**, even I get the same error 
i have tried **uninstalling an reinstall installing bardapi** but it didn't fix the error

**Reply by cumaron** [reply]
In my recent tests (27th of August) I found that the Bard API works, if 
1) you are logged into your Google account (and get the 1PSID token from the Google Bard website),
2) after you logged out and logged in again, you need to supply a new 1PSID token, which will be different from the previous login.

**Reply by Jaijith-guddemane** [reply]
@cumaron it worked for me, thank you
